products/bash/release-products.sh

```diff
@@ -93,6 +93,94 @@ chart_repo=ibm-entitled-charts
 helm repo add ${chart_repo} https://raw.githubusercontent.com/IBM/charts/master/repo/entitled/
 
 
+# --- Tracing registration ---------------------------------------------
+# First parameter is the namespace where tracing is installed
+# Second parameter is the namespace to register. I.e. mq if registering mq
+# Note before calling this tracing should be installed and the app to be registered should be installed
+function register_tracing_application {
+  TRACING_NAMESPACE=${1}
+  REG_NAMESPACE=${2}
+
+  echo "Registering app in ${REG_NAMESPACE} namespace for tracing"
+
+  echo "Waiting for tracing registration job to complete..."
+  for i in `seq 1 60`; do
+  	TRACING_JOB_STATUS=$(kubectl get pods -n ${REG_NAMESPACE} | grep -E -m1 '(odtracing|od-registration)' | awk '{print $3}')
+  	if [ "$TRACING_JOB_STATUS" == "Completed" ]; then
+  		echo "Tracing registration job is complete."
+  		break
+  	else
+      if [[ i -eq 60 ]]; then
+        echo "Tracing registration job failed, giving up after numerous attempts"
+        exit 1
+      fi
+  		echo "Waiting for tracing registration job to complete (Attempt $i of 60). Job status: $TRACING_JOB_STATUS"
+  		kubectl -n ${REG_NAMESPACE} get pods,jobs
+  		echo "Checking again in one minute..."
+  		sleep 60
+  	fi
+  done
+
+  echo "Waiting 1 minute for tracing registration to trickle through"
```

Comment:
Do we want to be funny here? :)

Reply:
Not sure what you're asking for!

Reply:
something like: Waiting for tracing registration to complete

Reply:
Ah, ok, I'm happy with "trickle through"